odl/contrib/datasets/trafos/tugraz.py

```diff
@@ -0,0 +1,244 @@
+    """Reconstruction operator for 4 channel MRI of a head.
+
+    The scan is a T2 weighted TSE scan of a healthy volunteer.
+
+    The reconstruction operator is the sum of squares of each channel.
+
+    Notes
+    -----
+    See the data source with DOI `10.5281/zenodo.800525`_ or the
+    `project webpage`_ for further information.
+
+    See Also
+    --------
+    mri_head_data_4_channel
+
+    References
+    ----------
+    .. _10.5281/zenodo.800525: https://zenodo.org/record/800525
+    .. _project webpage: http://imsc.uni-graz.at/mobis/internal/\
+platform_aktuell.html
+    """
+    # To get the same rotation as in the reference article
+    space = odl.uniform_discr(min_pt=[-115.2, -115.2],
+                              max_pt=[115.2, 115.2],
+                              shape=[256, 256],
+                              dtype=complex)
+
+    trafo = odl.trafos.FourierTransform(space)
+
+    return odl.ReductionOperator(odl.ComplexModulus(space) * trafo.inverse, 4)
+
+
+def mri_head_data_32_channel():
+    """Raw data for 32 channel MRI of a head.
+
+    The scan is a T2 weighted TSE scan of a healthy volunteer.
+
+    The data has been rescaled to fit approximately in [0, 1].
+
+    Notes
+    -----
+    See the data source with DOI `10.5281/zenodo.800527`_ or the
+    `project webpage`_ for further information.
+
+    See Also
+    --------
+    mri_head_inverse_32_channel
+
+    References
+    ----------
+    .. _10.5281/zenodo.800527: https://zenodo.org/record/800527
+    .. _project webpage: http://imsc.uni-graz.at/mobis/internal/\
+platform_aktuell.html
+    """
+    # TODO: Store data in some ODL controlled url
+    url = 'https://zenodo.org/record/800527/files/2_rawdata_brainT2_32ch.mat'
+    dct = get_data('2_rawdata_brainT2_32ch.mat', subset=DATA_SUBSET, url=url)
+
+    # Change axes to match ODL definitions
+    data = np.flip(np.swapaxes(dct['rawdata'], 0, -1) * 7e3, 2)
+
+    return data
+
+
+def mri_head_inverse_32_channel():
+    """Reconstruction operator for 32 channel MRI of a head.
+
+    The scan is a T2 weighted TSE scan of a healthy volunteer.
+
+    The reconstruction operator is the sum of squares of each channel.
+
+    Notes
+    -----
+    See the data source with DOI `10.5281/zenodo.800527`_ or the
+    `project webpage`_ for further information.
+
+    See Also
+    --------
+    mri_head_data_32_channel
+
+    References
+    ----------
+    .. _10.5281/zenodo.800529: https://zenodo.org/record/800527
+    .. _project webpage: http://imsc.uni-graz.at/mobis/internal/\
+platform_aktuell.html
+    """
+    # To get the same rotation as in the reference article
+    space = odl.uniform_discr(min_pt=[-115.2, -115.2],
+                              max_pt=[115.2, 115.2],
+                              shape=[256, 256],
+                              dtype=complex)
+
+    trafo = odl.trafos.FourierTransform(space)
+
+    return odl.ReductionOperator(odl.ComplexModulus(space) * trafo.inverse, 32)
```

Comment:
This doesn't agree with the doc since `ComplexModulus` takes the square root of the sum of squares. (across the board)